acceptance/tests/resource/service/ticket_5024_systemd_enabling_masked_service.rb

```diff
@@ -25,8 +25,9 @@
 
 agents.each do |agent|
   platform = agent.platform.variant
+  version = on(agent, facter('os.release.full')).stdout.chomp.to_i
```

Comment:
@skyamgarp A minor time efficiency thing, if we could run this version statement only when platform is ubuntu I think that would save to run this command for other platforms where it is not needed.

Reply:
LGTM. Thanks Swati.